src/test/java/org/opensearch/geospatial/ClusterSettingHelper.java

```diff
@@ -44,12 +49,28 @@ public Node createMockNode(Map<String, Object> configSettings) throws IOExceptio
         return new MockNode(baseSettings().build(), basePlugins(), configDir, true);
     }
 
-    private List<Class<? extends Plugin>> basePlugins() {
+    private Collection<PluginInfo> basePlugins() {
         List<Class<? extends Plugin>> plugins = new ArrayList<>();
         plugins.add(getTestTransportPlugin());
         plugins.add(MockHttpTransport.TestPlugin.class);
         plugins.add(TestGeospatialPlugin.class);
-        return plugins;
+
+        Collection<PluginInfo> pluginInfos = plugins.stream()
+            .map(
+                p -> new PluginInfo(
+                    p.getName(),
+                    "classpath plugin",
+                    "NA",
+                    Version.CURRENT,
+                    "1.8",
+                    p.getName(),
+                    null,
+                    Collections.emptyList(),
+                    false
```

Comment:
we need those settings just to mock the plugin info class for testing purposes, correct?